de.metas.adempiere.adempiere/base/src/main/java/de/metas/bpartner/service/impl/BPartnerDAO.java

```diff
@@ -1037,18 +1038,39 @@ public I_C_BPartner_Location retrieveBPartnerLocation(@NonNull final BPartnerLoc
 				.addOnlyActiveRecordsFilter();
 
 		queryBuilder.orderBy()
-				.addColumn(I_C_BP_Relation.COLUMNNAME_C_BP_Relation_ID);
+			.addColumn(I_C_BP_Relation.COLUMNNAME_C_BP_Relation_ID);
 
-		final I_C_BP_Relation billtoRelation = bpRelationQueryBuilder
+		final Stream<BPRelation> relationsStream = bpRelationQueryBuilder
```

Comment:
what's the point of having this outside of `if (query.getRelationBPartnerLocationId() != null)`